avro-fastserde/src/main/java/com/linkedin/avro/fastserde/coldstart/ColdPrimitiveFloatList.java

```diff
@@ -0,0 +1,22 @@
+package com.linkedin.avro.fastserde.coldstart;
+
+import com.linkedin.avro.api.PrimitiveFloatList;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+
+public class ColdPrimitiveFloatList extends GenericData.Array<Float> implements PrimitiveFloatList {
```

Comment:
javadoc - "implements PrimitiveFloatList by delegating to the normal (expensive) calls" or something

Reply:
Done.